src/common/components/game/AbilitiesTooltip.js

```diff
@@ -53,6 +55,7 @@ export default class AbilitiesTooltip extends Component {
         body={this.tooltipBody}
         refreshIntervalMs={50}
         preferPlace="right"
+        appendTarget={getGameAreaNode()}
```

Comment:
You had to do this in both the `AbilitiesTooltip` and the `CardTooltip` which is a bit of a code smell. Could the common part of these two Tooltips be extracted into its own component or helper function, or is that abstraction a lot of overhead? Maybe a wrapper of Popover with the common configuration.